scripts/openssl-build.bash

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
This documents the workaround for building with OpenSSL and is referenced by BUILD.md.